seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2653,6 +2653,47 @@
       "challengeType": 1
     },
     {
+      "id": "5690307fddb111c6084545d7",
+      "title": "Logical Order in If Else Statements",
+      "description": [
+        "Order is important in <code>if</code>, <code>else if</code> statements.",
+        "The loop is executed from top to bottom so you will want to be careful of what statement comes first.",
+        "Take these two functions as an example.",
+        "Heres the first:",
+        "<blockquote>function foo(x) {<br>  if (x < 1) {<br>    return \"Less than one\";<br>  } else if (num < 2) {<br>    return \"Less than two\";<br>  } else {<br>    return \"Greater than or equal to two\";<br>  }<br>}</blockquote>",
+        "And the second just switches the order of the statements:",
+        "<blockquote>function bar(x) {<br>  if (x < 2) {<br>    return \"Less than two\";<br>  } else if (num < 1) {<br>    return \"Less than one\";<br>  } else {<br>    return \"Greater than or equal to two\";<br>  }<br>}</blockquote>",
+        "While these two functions look nearly identical if we pass a number to both we get different outputs.",
+        "<blockquote>foo(0) // \"Less than one\"<br>bar(0) // \"Less than two\"</blockquote>",
+        "<h4>Instructions</h4>",
+        "Change the order of order of logic in the function so that it will return the correct statements in all cases."
```

Comment:
Doesn't **order of order of** sound redundant? I think something like "Change the logical order of if-else statements in the function..." is better. Otherwise everything is good.